### PR TITLE
Sci 1027 vcf sitesonly

### PR DESCRIPTION
**Pull request by nainathangaraj**
#dxapp.json
- [x] Upgrade to 14.04
- [x] Add help and extension to input spec.
- [x] In `details` field specify: `whatsNew`and `citation` (leave blank for citation.
- [x] In `details` consider specifying `upstreamAuthor` and `upstreamVersion` fields when applicable.
- [x] Use `regionalOptions` 
- [x] Set applicable timeout (Be relax) and restart policy in [`runSpec`](https://wiki.dnanexus.com/API-Specification-v1.0.0/IO-and-Run-Specifications).
- [x] Add subjob restartability when applicable.
- [x] `version` field, set proper [semantic versioning](https://semver.org/).
- [x] Setting `"openSource": true`. 
- [x] Run [dxapp linter](https://github.com/dnanexus/RoughER/blob/master/scripts/dxapp_lint.py) [[permalink](https://github.com/dnanexus/RoughER/blob/ecb3ea3f97870c020f93bf4723afc9f40a310e33/scripts/dxapp_lint.py)] on `dxapp.json`.
- [x] Use AssetDepend instead of execDepend
- [x] Add default instance type that works for a general use case.
- [x] add new input if applicable

#src/code.sh src/code.py
- [x] GATK tarball should be removed from the resource project (or make a new resource project), and the applet should require GATK tarball as a required input. This way, the users are enforced to provide their own tarball. This would avoid trouble from non-academic entities without GATK license to use your tarball.

# Readme.md
- [ ] Improve Readme file to contain applet function, use case, input/output description, underline execution if it’s a wrapper.
- [ ] Read documentation for sanity check. (Unpublished app URL will be: https://platform.dnanexus.com/app/app-xxxx)

# Test
- [x] Basic testing using given input in Box or expected input
- [x] Make a Pull Request to `master` in https://github.com/RitchieLab/dnanexus

# Production
- [ ] Co-test with Ritchie group
- [ ] Help Ritchie group to publish (Give dx command)
- [ ] Verify the versions that appear on the platform are the intended versions
- [ ] Set up demo workflow in public project.

### CODE REVIEW

**Comment by Arkarachai** [comment]
Except for the readme, everything else looks great. There is even a test which I'm impressed.
-IMO, we should keep the opensource=True. 
-I forgot to add to the checklist that we should remove authorized user, but you have done the right thing. 
-I'm not sure about the developer list. Maybe you are right that we should remove it.

**Comment by Arkarachai** [comment]
For what's new, I'm fine with either blank or the message you provide. Since this will be the first version of app, we can keep it blank. It's good to include the key though.

**Comment by nainathangaraj** [comment]
Thanks, Chai! I will update the dxapp.json with a blank "what's new" field now, and add more to the ReadMe.md. 

For the developers list, we can add them using the API call if required. Maybe, this is something that we can ask Marylyn and her group, if they would like to include the developers going forward.

**Comment by nainathangaraj** [comment]
@slzarate : Please take a look at the readme when you get a chance. Thanks!

File: sequencing/vcf_sitesonly/dxapp.json [hunk 1]
```diff
@@ -22,32 +23,33 @@
...
+                      "assetDepends": [{"id": "record-FJ3zf8j08zVZFzqYG88Pbk7z"}]},
+    "azure:westus": {"systemRequirements": {"*": {"instanceType": "azure:mem1_ssd1_x4"}},
+                      "assetDepends": [{"id": "record-FJ4058j9Vf9ZBBVB2Q7b3VQz"}]}
   },
-  "developers": ["user-tnperson", "user-johnwallace123", "user-auv13"],
-  "openSource" : true,
-  "authorizedUsers" : ["org-geisinger_affiliate", "org-geisinger", "org-support", "user-codushlaine", "user-semanti.mukherjee", "user-alexander.li", "user-shannon.bruse", "user-jan.freudenberg", "user-crisvh"]
-}
+    "details": {
+    "whatsNew": ""
```
Comment (yihchii) [inline sequencing/vcf_sitesonly/dxapp.json]: "whatsNew": "* 1.0.0 First version of the Get Sites-only VCF app."

**Review by yihchii — COMMENTED** [review]
LGTM. I have a minor comment about whatsnew.

File: sequencing/vcf_sitesonly/Readme.md [hunk 2]
```diff
@@ -1,21 +1,18 @@
 <!-- dx-header -->
-# VCF QC (DNAnexus Platform App)
+# Get Sites-only VCF
```
Comment (slzarate) [inline sequencing/vcf_sitesonly/Readme.md]: Get Sites-Only VCF

File: sequencing/vcf_sitesonly/Readme.md [hunk 3]
```diff
@@ -1,21 +1,18 @@
...
-TODO: This app directory was automatically generated by dx-app-wizard;
-please edit this Readme.md file to include essential documentation about
-your app that would be helpful to users. (Also see the
-Readme.developer.md.) Once you're done, you can remove these TODO
-comments.
+## What does this app output?
 
-For more info, see https://wiki.dnanexus.com/Developer-Portal.
--->
+This app outputs a site only VCF file(s) (`*.vcf.gz`) along with this index file. This output VCF file (`*.vcf.gz`) would contain the first eight columns of the original VCF file.
```
Comment (slzarate) [inline sequencing/vcf_sitesonly/Readme.md]: This app outputs a site-only VCF file(s) (`*.vcf.gz`) along with its index file. The output VCF file (`*.vcf.gz`) will contain the first eight columns of the original VCF file.

File: sequencing/vcf_sitesonly/Readme.md [hunk 4]
```diff
@@ -1,21 +1,18 @@
...
+
+## How does this app work?
+
+This app uses the cut command to get a sites-only vcf (1st 8 columns) of the VCF file (`*.vcf.gz`).
```
Comment (slzarate) [inline sequencing/vcf_sitesonly/Readme.md]: This app uses the `cut` command to get a sites-only subset (the first 8 columns) of the VCF file (`*.vcf.gz`).

**Pull request merged**